swarm/network/simulation/node_test.go

```diff
@@ -289,19 +289,21 @@ func TestUploadSnapshot(t *testing.T) {
 				HiveParams:   hp,
 			}
 			kad := network.NewKademlia(addr.Over(), network.NewKadParams())
+			b.Store(BucketKeyKademlia, kad)
 			return network.NewBzz(config, kad, nil, nil, nil), nil, nil
 		},
 	})
 	defer s.Close()
 
 	nodeCount := 16
 	log.Debug("Uploading snapshot")
-	err := s.UploadSnapshot(fmt.Sprintf("../stream/testing/snapshot_%d.json", nodeCount))
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
```

Comment:
Be consistent, please. Here you write `time.Second*30`; later you write  `3*time.Minute`. The latter reads better for me; I just prefer consistency even more.